src/hooks/useSigner.ts

```diff
@@ -1,49 +1,30 @@
-import { useState, useEffect } from 'react';
-import { ethers } from 'ethers';
+import * as React from 'react';
+import { type WalletClient, useWalletClient } from 'wagmi';
+import { providers } from 'ethers';
 
-type Window = {
-  ethereum: ethers.providers.ExternalProvider;
-};
+function walletClientToSigner(walletClient: WalletClient) {
+  const {
+    account,
+    chain,
+    transport,
+  } = walletClient;
 
-/**
- * React hook that gets signer from window.ethereum
- * @returns ethers signer
- */
-export const useSigner = () => {
```

Comment:
That function seems to be still used in [hopr-admin](https://github.com/hoprnet/hopr-admin/tree/add-safe-signatures-required-section)/[src](https://github.com/hoprnet/hopr-admin/tree/add-safe-signatures-required-section/src)/[steps](https://github.com/hoprnet/hopr-admin/tree/add-safe-signatures-required-section/src/steps)/[install-node](https://github.com/hoprnet/hopr-admin/tree/add-safe-signatures-required-section/src/steps/install-node)
/nodeAddress.tsx